ush/config_defaults.yaml

```
@@ -1461,10 +1461,43 @@ task_make_orog:
       rundir: "{{ task_make_orog.rundir }}/shave4"
   filtered_output: "{{ task_make_orog.filter_topo.rundir }}/{{ task_make_orog.filter_topo.config.filtered_orog }}"
 task_make_sfc_climo:
-  KMP_AFFINITY_MAKE_SFC_CLIMO: "scatter"
-  OMP_NUM_THREADS_MAKE_SFC_CLIMO: 1
-  OMP_STACKSIZE_MAKE_SFC_CLIMO: "1024m"
   SFC_CLIMO_DIR: '{{ workflow.EXPTDIR + "/sfc_climo" if rocoto.tasks.get("task_make_sfc_climo") else "" }}'
+  sfc_climo_gen:
+    execution:
+      batchargs:
+        export: NONE
+        nodes: 1
+        tasks_per_node: 24
+        stdout: "{{ user.HOMEdir }}/scripts"
+        walltime: 00:02:00
+      envcmds:
+        - module use {{ user.HOMEdir }}/modulefiles
+        - module load build_{{ user.MACHINE|lower() }}_{{ workflow.COMPILER }}
+      executable: "{{ user.EXECdir }}/sfc_climo_gen OR {{ user.HOMEdir }}/{{ workflow.EXEC_SUBDIR }}/sfc_climo_gen"
```

```suggestion
      executable: "{{ user.EXECdir }}/sfc_climo_gen"
```
I was a bit skeptical of this syntax and see that your run command in the run script for one of your tests is running this:


```
time srun --export=ALL /scratch1/BMC/zrtrr/Naureen.Bharwani/SRW_uwtools_integration/ufs-srweather-app/exec/sfc_climo_gen OR /scratch1/BMC/zrtrr/Naureen.Bharwani/SRW_uwtools_integration/ufs-srweather-app/exec/sfc_climo_gen
```

I am a little stumped by bash right now. Not sure how that succeeded (but it did) according to the error handling:

```
test $? -eq 0 && touch runscript.sfc_climo_gen.done
```

---

Wow that's interesting. This was when I was first starting out and I commented this out for reference for myself to say one of these should pass the test cases, but was unsure which to do with. I'm also surprised this passed.